client/components/pages/users-list-page/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { Container, TableCell, TableRow } from '@material-ui/core';
+import moment from 'moment';
+import React, { memo, useMemo } from 'react';
+import { useQuery } from 'react-apollo';
+import { sortBy } from 'sort-by-typescript';
+
+import { GET_USERS_CURSOR } from '_apollo/queries';
+import { ColumnProps } from '_client/components/common/table';
+import { SortDirection, User, UsersDataCursor } from '_client/types';
+import { ContainerBox, PageContainer, Table } from '_components/common';
+import RemoveUserButton from '_components/pages/users-list-page/removeUserButton';
+import UserRoleSelect from '_components/pages/users-list-page/userRoleSelect';
```

Comment:
```suggestion
import RemoveUserButton from './RemoveUserButton';
import UserRoleSelect from './UserRoleSelect';
```

и поменяй названия этих файлов, чтоб были CamelCase.tsx